cos-kernel

```diff
@@ -184,53 +214,55 @@ parse_args() {
 	local args
 
 	if ! args=$(getopt \
-			--options "ac:hi:pvVx" \
-			--longoptions "all config: help instdir: print verbose no-xtract" \
+			--options "ac:hi:prvVx" \
+			--longoptions "all config: help instdir: print no-remove verbose no-xtract" \
 			-- "$@"); then
 		# getopt has printed an appropriate error message.
 		exit 1
 	fi
 	eval set -- "${args}"
 
 	while [[ "${#}" -gt 0 ]]; do
-		case "${1}" in
+		case "$1" in
 		-a|--all)
```

Comment:
Hum, I thought `${1}` and `$1` should have no difference in this case. And `${1}` seems pretty good to me. Is there any benefit in changing it in this case?

Reply:
They are the same.  I removed the braces to comply with bash style guide: "Don't brace-quote single character shell specials / positional parameters".